ViewModels/BluetoothViewModel.cs

```diff
@@ -9,21 +9,66 @@
 using EnlightenMAUI.Models;
 using EnlightenMAUI.Common;
 
+//using Android.Hardware.Usb;
+//using Android.Content;
+using LibUsbDotNet.Main;
+using Android.Content;
+using Android.Hardware.Usb;
+using Android.App;
+using Android.Nfc;
+using Microsoft.Maui.Controls.PlatformConfiguration;
+using Microsoft.Maui;
+using Xamarin.Google.Crypto.Tink.Subtle;
```

Comment:
That is the funniest package name I've ever read.

Reply:
it is unfortunately unneeded, but I agree